industrial_ci/src/folding/none.sh

```diff
@@ -17,8 +17,10 @@
 
 function  ici_start_fold() {
     shift 3
+    ici_color_output $ANSI_BLUE ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
 }
 
 function  ici_end_fold() {
     shift 4
+    ici_color_output "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
```

Comment:
color is set beforehand..
```suggestion
    echo "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
```